lab_1_classify_by_unigrams/main.py

```diff
@@ -19,6 +31,16 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     :param tokens: a list of tokens
     :return: a dictionary with frequencies
     """
+    if not isinstance(tokens, list):
+        return None
+    frequency_dict = {}
+    tokens_number = len(tokens)
+    for token in tokens:
+        if not isinstance(token, str):
+            return None
+        token_frequency = tokens.count(token)
```

Comment:
what if tokens contains a token not once - you count the same value multi times

Reply:
fixed